src/stylesheets/components/_navigation.scss

```diff
@@ -33,42 +41,38 @@
 
 .navigation--letter,
 .navigation--number {
-	@include navigation(none, 0, 0);
-	margin-bottom: $base-spacing-unit-small;
+	@include navigation(0, $base-spacing-unit-tiny);
 
 	li {
 		text-align: center;
 		text-transform: capitalize;
 		padding: 0;
-
-		a,
-		span {
-			display: block;
-			min-width: 23.5px;
-		}
+	}
+	a,
+	span {
```

Comment:
li.active will have a child **span** instead of an **a** tag, thereby loosing padding, do we need to add padding to the span?